documentation/src/css/custom.css

```
@@ -1,95 +1,286 @@
+  height: 3px;
+  background: var(--gradient-secondary);
+  opacity: 0.8;
 }
 
 .performance-metric:hover {
+  background: var(--gradient-metric-hover);
+  transform: translateY(-5px);
+  box-shadow: 
+    var(--shadow-red-lg),
+    inset 0 0 15px rgba(0, 0, 0, 0.3);
   background: rgba(229, 57, 57, 0.15);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)